community/cython/APKBUILD

```diff
@@ -45,11 +45,20 @@ package() {
 _py2() {
 	cd "$builddir"-py2
 	_py python2
+
```

Comment:
Using `local f` is good practice to avoid leaking the `f` variable into later executions.

Reply:
Implemented.